doc/docusaurus/docs/reference/CIP-0069-changes.md

```diff
@@ -0,0 +1,177 @@
+---
+sidebar_position: 27
+---
+
+# Understanding CIP-0069 changes
+
+## Overview
+
+Cardano Improvement Proposal 69 [(CIP-0069)](https://cips.cardano.org/cip/CIP-0069) introduces significant changes to how Plutus scripts are written and executed. 
+It streamlines the development and functionality of Plutus scripts by unifying the arguments that are provided to different types of scripts. 
+Specifically, it removes the datum argument, which has been a source of complexity and limitations in decentralized application (DApp) development on the Cardano blockchain.
+These changes also reduce the chances of attacks succeeding. 
+
+We will review these changes and take a look at how to implement them.
+
+## 1. Unified signature for all Plutus scripts
+
+### What changed
+
+Previously, different types of scripts (like spending or minting) had different structures, making it challenging to create multi-purpose scripts. 
+CIP-0069 introduces a unified signature for all Plutus scripts, regardless of their purpose.
+This unification simplifies the Plutus codebase and allows for more flexible script design.
+
+### Implementation of new signature
+To implement this change, it is necessary to update all scripts to use this new signature:
+```haskell
+ScriptArgs -> ScriptContext -> ()
```

Comment:
> ScriptArgs -> ScriptContext -> ()

Another CIP (117) has changed the last type from unit () to BuiltinUnit so if somebody tries this signature its going to fail because deployed node expects BuiltinUnit

Reply:
Great to know about that. Thanks.

Reply:
Yes I agree with @Unisay , I think it is better to address CIP-69 and CIP-117 together in one sweep.

So the signature after applying both of these 2 CIPs will be in the end:

`ScriptArgs -> ScriptContext -> BuiltinUnit`